packages/core/src/AvatarGroup/AvatarGroup.styles.tsx

```diff
@@ -10,11 +10,32 @@ export const { staticClasses, useClasses } = createClasses("HvAvatarGroup", {
       border: `2px solid ${theme.colors.atmo2}`,
       boxSizing: "content-box",
     },
+    "&$row": {
```

Comment:
`row`, `column`, `highlight` are applied on the same element as `root`, can't this be moved to the respective block? It simplifies the code and reduces (the already strong) specificity